lib/context.dart

```diff
@@ -0,0 +1,7 @@
+class RetroContext {
+  static const String versionName = '0.2.0';
+  static const int versionCode = 0;
+  static const String branch = 'main';
+  static const String commitHash = '66e4c03';
```

Comment:
where is this being used? hardcoding a commit sha into code doesn't feel good (especially because it's already wrong and will definitely be wrong after squashing this)

Reply:
The problem is that, there is not a real way to get this because of how its compiled there is no cmake